cask-repair

```diff
@@ -256,7 +256,7 @@ function appcast_checkpoint_change {
 function delete_created_branches {
   local local_branches remote_branches
 
-  for dir in "${caskroom_taps_dir}"/*; do
+  for dir in "${caskroom_taps_dir}/homebrew-${caskroom_taps}"; do
```

Comment:
This shouldn’t be an issue. It’s true that it’ll now cycle through the other non-cask taps, but it’ll still only affect `cask-repair_*` branches and only in your own fork.

Still, not harm in being more precise.

But as it is, it will not cycle through the `casktoom_taps` array and will only give one directory. The correct form should be `for dir in "${caskroom_taps_dir}/homebrew-cask"*; do`.